app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -269,33 +269,35 @@ private fun WallpaperThumbnailItem(
 }
 
 @Composable
+@Suppress("MagicNumber")
 private fun WallpaperLogoSwitch(
     checked: Boolean,
     onCheckedChange: (Boolean) -> Unit
 ) {
-    Column(
-        modifier = Modifier.padding(horizontal = 12.dp, vertical = 16.dp),
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier
+            .padding(horizontal = 12.dp, vertical = 16.dp)
+            .fillMaxWidth()
     ) {
-        Row(
-            horizontalArrangement = Arrangement.SpaceBetween,
-            modifier = Modifier.fillMaxWidth()
-        ) {
-            Text(
-                text = stringResource(R.string.wallpaper_tap_to_change_switch_label_1),
-                color = FirefoxTheme.colors.textPrimary,
-                fontSize = 18.sp,
-                modifier = Modifier.padding(start = 4.dp)
-            )
-            Switch(
-                checked = checked,
-                onCheckedChange = onCheckedChange,
-                colors = SwitchDefaults.colors(
-                    checkedThumbColor = FirefoxTheme.colors.formSelected,
-                    checkedTrackColor = FirefoxTheme.colors.formSurface,
-                    uncheckedTrackColor = FirefoxTheme.colors.formSurface
-                )
+        Text(
+            text = stringResource(R.string.wallpaper_tap_to_change_switch_label_1),
+            color = FirefoxTheme.colors.textPrimary,
+            fontSize = 18.sp,
+            modifier = Modifier
+                .padding(start = 4.dp)
+                .weight(0.8f)
```

Comment:
Just to highlighting, this change as the this maybe a bit difficult to read.

Reply:
Alternatively, just the `Text()` could have the weight, and it will always grow to the maximum remaining horizontal space.

Reply:
👍🏽

Reply:
Addressed!